core/services/mongodb-remote/src/main/java/com/mongodb/stitch/core/services/mongodb/remote/sync/internal/DataSynchronizer.java

```diff
@@ -654,6 +789,8 @@ private void syncLocalToRemote() {
           continue;
         }
 
+        // i. Retrieve the synthetic change event for this local document in the local config
```

Comment:
[nit] We may want to be careful about using the word `synthetic` here. Local events exist in a transient state, but are arguably real. Synthetic as term makes more sense when we are generating remote events based on conflict, and do not have actual data about what the remote change event was. I feel it's important that we do not conflate terminology here.

Reply:
Alright, removing the word synthetic, and updating the spec as well.